### ISSUE TITLE
Latest update has broken shopping list

### ISSUE THREAD

**Issue by sun2everyone** [EVEIPH/EVE-IPH#153]
Hello!
Win8.1 x64
Whenever I try to open non-empty shopping list error occures:

```
An Unhandled Exception has occured and EVE Isk per Hour will now close.

Please fill out the following information so I can reproduce the bug

What is your Operating System? 
What tab or screen did the error occur? 
What are the steps to reproduce the Error? 
Web link to a screenshot of your error: 
In addition to a screenshot, copy the data below and send to developer.

Source: Microsoft.VisualBasic
Message: Conversion from string "" to type 'Double' is not valid.
Raw Error Text: System.InvalidCastException: Conversion from string "" to type 'Double' is not valid. ---> System.FormatException: Входная строка имела неверный формат.
   в Microsoft.VisualBasic.CompilerServices.Conversions.ParseDouble(String Value, NumberFormatInfo NumberFormat)
   в Microsoft.VisualBasic.CompilerServices.Conversions.ToDouble(String Value, NumberFormatInfo NumberFormat)
   --- Конец трассировки внутреннего стека исключений ---
   в Microsoft.VisualBasic.CompilerServices.Conversions.ToDouble(String Value, NumberFormatInfo NumberFormat)
   в Microsoft.VisualBasic.CompilerServices.Conversions.ToDouble(String Value)
   в EVE_Isk_per_Hour.frmShoppingList.LoadFormStats()
   в EVE_Isk_per_Hour.frmShoppingList.RefreshLists()
   в EVE_Isk_per_Hour.frmMain.ShowShoppingList()
   в EVE_Isk_per_Hour.frmMain.mnuSelectionShoppingList_Click_1(Object sender, EventArgs e)
   в System.Windows.Forms.ToolStripItem.RaiseEvent(Object key, EventArgs e)
   в System.Windows.Forms.ToolStripMenuItem.OnClick(EventArgs e)
   в System.Windows.Forms.ToolStripItem.HandleClick(EventArgs e)
   в System.Windows.Forms.ToolStripItem.HandleMouseUp(MouseEventArgs e)
   в System.Windows.Forms.ToolStripItem.FireEventInteractive(EventArgs e, ToolStripItemEventType met)
   в System.Windows.Forms.ToolStripItem.FireEvent(EventArgs e, ToolStripItemEventType met)
   в System.Windows.Forms.ToolStrip.OnMouseUp(MouseEventArgs mea)
   в System.Windows.Forms.ToolStripDropDown.OnMouseUp(MouseEventArgs mea)
   в System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   в System.Windows.Forms.Control.WndProc(Message& m)
   в System.Windows.Forms.ScrollableControl.WndProc(Message& m)
   в System.Windows.Forms.ToolStrip.WndProc(Message& m)
   в System.Windows.Forms.ToolStripDropDown.WndProc(Message& m)
   в System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)
   в System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   в System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
Error Tracking: 
```

**Comment by Zifrian** [comment]
Ok, any specific items you add to it or just any item?

**Comment by sun2everyone** [comment]
Any item I tried, whenever it is an owned blueprint or not. The same happens if I have previously opened empty shopping list (empty window opens fine), and then add any item. 
So, there are two ways to reproduce the error I found so far:
1. Add any item to shopping list via "Add to shopping list" button, and then click "View shopping list" menu.
2. Open empty shopping list window via "View shopping list" menu, and then click "Add to shopping list" button.

**Comment by sun2everyone** [comment]
Error example:
![iph_err](https://user-images.githubusercontent.com/33718800/85583621-1802ee80-b647-11ea-86bb-add2237b3cd3.jpg)

**Comment by Zifrian** [comment]
Thanks I’ll look at it tonight.

**Comment by Zifrian** [comment]
This is fixed in build 4.0.7481.1002
